pkg/services/featuremgmt/registry.go

```diff
@@ -1240,6 +1240,12 @@ var (
 			Owner:        grafanaSharingSquad,
 			FrontendOnly: true,
 		},
+		{
+			Name:        "datasourceProxyDisableRBAC",
```

Comment:
Makes sense, and I didn't know you could enable it by default, so that's why I went with the inverse logic of naming it "Disable".

Looking at the allowed stages, I am trying to decide between `FeatureStageGeneralAvailability` and `FeatureStagePublicPreview`. Do you think there's much of a difference for this case?

Reply:
Ah, sorry, of course you don't want to enable it by default if it's a disable FT 🤦 I think those two are very similar, I would maybe go with `FeatureStageGeneralAvailability`, as technically the "feature" that you would enable by switching it on is the old behaviour, which has been GA for ages. Also make sure to hide this from the docs (`HideFromDocs`), as this is probably for internal use only.

Reply:
Nice, thanks for the clarification on that! Makes sense, updated here - a2eec05a25c